packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -324,6 +332,30 @@
   font-size: $size-body-caption;
 }
 
+.heading1Size {
+  font-size: $size-heading-1;
+}
+
+.heading2Size {
+  font-size: $size-heading-2;
+}
+
+.heading3Size {
+  font-size: $size-heading-3;
+}
+
+.heading4Size {
+  font-size: $size-heading-4;
+}
+
+.heading5Size {
+  font-size: $size-heading-5;
+}
+
+.heading6Size {
+  font-size: $size-heading-6;
+}
```

Comment:
having `heading*` styles but not the ability to actually add `h*` tags in the DOM feels strange to me - even if its just rhetoric in the CSS selectors here (they could just as easily be named something else). See discussion above